Wizualizacja gry/Obstacle/ObstacleClass.hpp

```diff
@@ -0,0 +1,29 @@
+//
+//  ObstacleClass.hpp
+//  TankBotFight
+//
+//  Created by Emil Panecki on 10/06/2021.
+//  Copyright © 2021 Emil Panecki. All rights reserved.
+//
+
+#ifndef ObstacleClass_hpp
```

Comment:
Maybe it would be better to use 
```
 #pragma once
 ```
 ? It's the same but 1 line instead of 3

Reply:
For me no difference.